### PR TITLE
Removing some old signups and phoenixLegacy related code.

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?

This PR removes some old `SignupController` methods, and a few no longer used methods in the `PhoenixLegacy` service.

### Any background context you want to provide?

Cleaning things! 🛁

### What are the relevant tickets/cards?

Refs [Pivotal ID #160602918](https://www.pivotaltracker.com/story/show/160602918)

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: app/Http/Controllers/SignupController.php [hunk 1]
```diff
@@ -53,27 +53,4 @@ public function store(Request $request)
...
-    /**
-     * Update the specified resource in storage.
-     *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  int  $id
-     * @return \Illuminate\Http\Response
-     */
-    public function update(Request $request, $id)
-    {
-        return '@todo update signup';
```
Comment (DFurnes) [inline app/Http/Controllers/SignupController.php]: The best kind of `@todo`! 🔥

File: routes/web.php [hunk 2]
```diff
@@ -69,4 +69,4 @@
 $router->resource('next/referrals', 'ReferralController', ['only' => ['store']]);
 
 // Signups
-$router->resource('next/signups', 'SignupController', ['except' => ['create', 'edit', 'destroy']]);
+$router->resource('next/signups', 'SignupController', ['except' => ['create', 'edit', 'destroy', 'show', 'update']]);
```
Comment (DFurnes) [inline routes/web.php]: Want to change this to `'only' => ['index', 'store']` for readability?

File: app/Http/Controllers/SignupController.php (hunk 1 above)
Comment (weerd) [reply app/Http/Controllers/SignupController.php]: hahahahahahahaha! 😂

File: routes/web.php (hunk 2 above)
Comment (weerd) [reply routes/web.php]: ah, yeah, makes sense!

**Pull request merged**